### PR TITLE
chore: pyjwt is an internal implementation detail it looks like

### PR DESCRIPTION
**Pull request by c0state**
The previous `pyjwt` property was internal only it looks like, so making more explicit with `__` prefix

### CODE REVIEW

**Comment by sloria** [comment]
will this be a major version bump, too?

File: flask_resty/jwt.py [hunk 1]
```diff
@@ -44,10 +44,10 @@ def get_credentials_from_token(self, token):
         return payload
 
     def decode_token(self, token):
-        return self.pyjwt.decode(token, **self.get_jwt_decode_args())
+        return self.__pyjwt.decode(token, **self.get_jwt_decode_args())
 
     @property
-    def pyjwt(self):
+    def __pyjwt(self):
```
[the remark this answers is not in the document]
Comment (c0state) [reply flask_resty/jwt.py:50]: Hmm, I think you're right. I actually used that first but then thought double underscore was better.

[the remark this answers is not in the document]
Comment (c0state) [reply flask_resty/jwt.py:50]:
```suggestion
    def _pyjwt(self):
```

File: flask_resty/jwt.py [hunk 2]
```diff
@@ -44,10 +44,10 @@ def get_credentials_from_token(self, token):
         return payload
 
     def decode_token(self, token):
-        return self.pyjwt.decode(token, **self.get_jwt_decode_args())
+        return self.__pyjwt.decode(token, **self.get_jwt_decode_args())
```
Comment (c0state) [inline flask_resty/jwt.py:47]:
```suggestion
        return self._pyjwt.decode(token, **self.get_jwt_decode_args())
```

**Comment by c0state** [comment]
> will this be a major version bump, too?

Yeah I was thinking technically that's the case here also.

**Pull request merged**